app/src/main/java/edu/mit/mitmobile2/news/activities/NewsStoryActivity.java

```diff
@@ -0,0 +1,90 @@
+package edu.mit.mitmobile2.news.activities;
+
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.webkit.WebView;
+import android.widget.ImageView;
+
+import com.squareup.picasso.Picasso;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import edu.mit.mitmobile2.Constants;
+import edu.mit.mitmobile2.MITActivity;
+import edu.mit.mitmobile2.R;
+import edu.mit.mitmobile2.news.models.MITNewsStory;
+import edu.mit.mitmobile2.news.utils.NewsUtils;
+import timber.log.Timber;
+
+public class NewsStoryActivity extends MITActivity {
+
+    @InjectView(R.id.story_web_view)
+    WebView storyWebView;
+    @InjectView(R.id.story_image_view)
+    ImageView storyImageView;
+
+    private MITNewsStory story;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_news_story);
+        ButterKnife.inject(this);
+
+        DisplayMetrics displayMetrics = getResources().getDisplayMetrics();
+
+        story = getIntent().getParcelableExtra(Constants.News.STORY);
+
+        Picasso.with(this).load(story.getCoverImage().getRepresentations().get(0).getUrl())
```

Comment:
I built 3 shortcut functions in the story model to get the 3 different size image urls for the cover image, use getOriginalCoverImageUrl() to get the url for this one